### PR TITLE
feature/AT-RESOURCES-29_Update_Log4j2

### PR DESCRIPTION
**Pull request by Bwore**
The correct format of the date displayed by Log4j2 is
YYYY-MM-dd

### CODE REVIEW

**Review by rodrigo-velasco — APPROVED** [review]

File: src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-DD HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n
```
Comment (ivanflores15) [inline src/main/resources/log4j2.properties]: Pattern is missing the closing bracket "}". 
Property "%msg" is duplicated, replace value of context key with "default".

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

File: src/main/resources/log4j2.properties [hunk 2]
```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default}
```
Comment (ivanflores15) [inline src/main/resources/log4j2.properties]: it's missing %n at end of line in order to have logs in different lines. please take a look on it.

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

File: src/main/resources/log4j2.properties (hunk 2 above)
Comment (Bwore) [reply src/main/resources/log4j2.properties]: Yes, sorry

File: src/main/resources/log4j2.properties [hunk 3]
```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default} 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default%n}
```
Comment (ivanflores15) [inline src/main/resources/log4j2.properties]: %n It's in wrong place you will get something like
**{logs
}{logs
}**
%n should be at end of log ( ... "context" : default}%n ) in that way logs are: 
**{logs}
{logs}**

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

File: src/main/resources/log4j2.properties (hunk 3 above)
Comment (Bwore) [reply src/main/resources/log4j2.properties]: Sorry, I didn't see that in my test

**Review by ivanflores15 — APPROVED** [review]

**Pull request merged**